web-client/iron-remote-desktop/src/services/PublicAPI.ts

```diff
@@ -74,7 +72,7 @@ export class PublicAPI {
             connect: this.connect.bind(this),
             setScale: this.setScale.bind(this),
             onSessionEvent: (partialObserver) => {
-                this.remoteDesktopService.sessionObserver.subscribe(partialObserver);
+                this.remoteDesktopService.sessionEventObservable.subscribe(partialObserver);
```

Comment:
naming: Similar, I guess this is a callback?

Reply:
Fixed